src/components/CurrentUser/CurrentUser.jsx

```diff
@@ -1,12 +1,83 @@
 import React from 'react';
+import PropTypes from 'prop-types';
+import { getUser } from '../../api/api';
 import './CurrentUser.scss';
 
-export const CurrentUser = () => (
-  <div className="CurrentUser">
-    <h2 className="CurrentUser__title"><span>Selected user: 2</span></h2>
+export class CurrentUser extends React.Component {
+  state = {
+    user: null,
+    errorMessage: '',
+  }
 
-    <h3 className="CurrentUser__name">Ervin Howell</h3>
-    <p className="CurrentUser__email">Shanna@melissa.tv</p>
-    <p className="CurrentUser__phone">010-692-6593 x09125</p>
-  </div>
-);
+  componentDidMount() {
+    const { userId } = this.props;
+
+    if (userId > 0) {
+      this.getUser(userId);
+    }
+  }
+
+  componentDidUpdate(prevProps) {
+    const { userId } = this.props;
+
+    if (prevProps.userId === userId) {
+      return;
+    }
+
+    this.getUser(userId);
+  }
+
+  getUser = async(id) => {
+    try {
+      const currentUser = await getUser(id);
+
+      this.setState({ user: currentUser });
+    } catch (error) {
+      this.setState({ errorMessage: error });
+    }
+  }
+
+  render() {
+    const { user, errorMessage } = this.state;
+    const { userId, clearUser } = this.props;
+
+    if (!user) {
+      return null;
+    }
+
+    return (
+      <div className="CurrentUser">
+        <h2 className="CurrentUser__title">
+          {!errorMessage ? user && (
```

Comment:
```suggestion
          {!errorMessage && user ? (
```
You check if there's no error and `user` is not null and then render either user's id or error